akka-serialization-jackson/src/test/scala/doc/akka/serialization/jackson/SerializationDocSpec.scala

```diff
@@ -101,6 +101,27 @@ object SerializationDocSpec {
     #//#several-config
   """
 
+  val configManifestless = """
+    #//#manifestless
+    akka.actor {
+      serializers {
+        jackson-json-event   = "akka.serialization.jackson.JacksonJsonSerializer"
+      }
+      serialization-identifiers {
+        jackson-json-event = 9001
+      }
+      serialization-bindings {
+        "com.myservice.MyEvent" = jackson-json-event
+      }
```

Comment:
Let's talk about how to do a rolling update, for example if started out with ordinary `jackson-json` binding and then switching over to this.

We have described a procedure for switching serializer in https://doc.akka.io/docs/akka/current/additional/rolling-updates.html#from-java-serialization-to-jackson

Do I understand it correctly that the multi-step procedure can work for this scenario if `serialization-bindings` is not defined in the first roll out, but instead using the `deserialization-type`?

Reply:
Yes, that's correct. I hadn't considered the rolling update scenario, but setting `deserialization-type` with no `serialization-bindings` would definitely be needed and would be the right approach. It may make sense to just always use `deserialization-type`, makes it explicit and decreases the surface area for a mistake.